src/server/components/lists/helpers.ts

```diff
@@ -228,6 +238,17 @@ export function formatCountryParam(country: string): string {
     if (countryName === "Northern Cyprus") {
       countryName = "northern Cyprus";
     }
+    if (countryName === "Cote D Ivoire") {
+      countryName = "Côte d'Ivoire";
+    }
+    countryName = restoreSpecialCharacter(".", country, countryName);
+    countryName = restoreSpecialCharacter(",", country, countryName);
+    countryName = restoreSpecialCharacter("-", country, countryName);
+    countryName = restoreSpecialCharacter("ã", country, countryName);
+    countryName = restoreSpecialCharacter("é", country, countryName);
+    countryName = restoreSpecialCharacter("í", country, countryName);
+    countryName = restoreSpecialCharacter("ç", country, countryName);
+    countryName = restoreSpecialCharacter("ô", country, countryName);
```

Comment:
This works fine as it is but I guess if we had time to refactor it might look cleaner with an array of special characters in a variable and then loop over that array to restore the special characters.

Reply:
thanks, makes sense.  Refactored accordingly.